packages/@okta/i18n/package.json

```diff
@@ -1,39 +1,106 @@
 {
   "_args": [
     [
-      {
-        "name": "@okta/i18n",
-        "raw": "@okta/i18n@37.3.0",
-        "rawSpec": "37.3.0",
-        "scope": "@okta",
-        "spec": "37.3.0",
-        "type": "version"
-      },
-      "/home/vagrant/okta/okta-signin-widget"
+      "@okta/i18n@37.5.0-alpha.4",
+      "/Users/mauricio.castillosilva/okta/okta-signin-widget"
     ]
   ],
-  "_from": "@okta/i18n@37.3.0",
-  "_id": "@okta/i18n@37.3.0",
+  "_from": "@okta/i18n@37.5.0-alpha.4",
+  "_id": "@okta/i18n@37.5.0-alpha.4",
   "_inCache": true,
   "_installable": true,
   "_location": "/@okta/i18n",
-  "_phantomChildren": {},
+  "_phantomChildren": {
+    "abbrev": "1.1.1",
```

Comment:
what are these?

Reply:
good question, I don't know. There is nothing new in i18n repo that I can see.... will try to figure out. Do you think I should remove them manually?